tests/e2e/utils/kubeflow_installation.py

```diff
@@ -236,7 +236,7 @@ def install_alb_controller(cluster_name):
             --set clusterName={cluster_name} \
             --set serviceAccount.create=false \
             --set serviceAccount.name=aws-load-balancer-controller \
-            --version v1.4.3"
+            --version v1.4.8"
```

Comment:
Shouldn't this be 7?

Reply:
https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/v2.4.7/helm/aws-load-balancer-controller/Chart.yaml#L4

Reply:
Ah ok